frame/election-provider-support/src/lib.rs

```diff
@@ -249,7 +253,7 @@ pub trait ElectionDataProvider<AccountId, BlockNumber> {
 
 #[cfg(feature = "std")]
 impl<AccountId, BlockNumber> ElectionDataProvider<AccountId, BlockNumber> for () {
-	const MAXIMUM_VOTES_PER_VOTER: u32 = 0;
+	type MaximumVotesPerVoter = ConstU32<0>;
```

Comment:
```suggestion
	type MaximumVotesPerVoter = frame_support::pallet_prelude::ConstU32<0>;
```

opinionated suggestion:  I always prefer code to impose the less amount of global imports, like in this case because it is already feature gates, and even without it it is not a fundamental part of this crate.

Reply:
Very good point, amended